src/placeos-frontend-loader/api/remote/remote.cr

```diff
@@ -61,9 +78,9 @@ module PlaceOS::FrontendLoader
       File.delete(temp_tar_name)
     end
 
-    def get_hash(hash : String, repository_uri : String, tag : String, branch : String)
+    def get_hash(hash : String, repository_uri : String, tag, branch : String)
       if hash == "HEAD" || hash.nil?
-        if tag != "latest"
+        if (!tag.nil? && tag != "latest")
```

Comment:
What is `"latest"`? I'm not aware of that value as a standard value - except maybe with docker containers.